docs/chart-design-and-style/design-and-style.md

```diff
@@ -176,4 +175,4 @@ In Highcharts, animation is enabled or disabled in two separate places.
 The animation can either be set as a boolean or a configuration object. If true, it will use the "swing" jQuery easing and a duration of 500 ms. If used as a configuration object, the following properties are supported:
 
 *   duration - The duration of the animation in milliseconds.
-*   easing - When using jQuery as the general framework, the easing can be set to linear or swing. More easing functions are available with the use of jQuery plug-ins, most notably the jQuery UI suite. See the [jQuery docs](https://api.jquery.com/animate/). When using MooTools as the general framework, use the property name transition instead of easing.
\ No newline at end of file
+*   easing - When using jQuery as the general framework, the easing can be set to linear or swing. More easing functions are available with the use of jQuery plug-ins, most notably the jQuery UI suite. See the [jQuery docs](https://api.jquery.com/animate/). When using MooTools as the general framework, use the property name transition instead of easing.
```

Comment:
This one is outdated - jQuery and particularly MooTools is history, so we should take this opportunity to fix it.
```suggestion
*   easing - Custom easing can be applied to all animation. See [Easing Functions](https://easings.net/) for a nice overview over some possible functions. Easing functions can either be passed directly, or registered on the `Math` object and referenced by name. See [chart/animation-easing](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/highcharts/chart/animation-easing) for a live demo. The default easing is `easeInOutSine`.
```